tools/SchemaManager/Commands/ApplyCommand.cs

```diff
@@ -114,12 +114,12 @@ await Policy.Handle<SchemaManagerException>()
                             {
                                 Console.WriteLine(string.Format(Resources.RetryCurrentVersions, attemptCount++, RetryAttempts));
                             })
-                        .ExecuteAsync(() => ValidateInstancesVersion(schemaClient, executingVersion));
+                        .ExecuteAsync(() => ValidateInstancesVersion(schemaClient, executingVersion, cancellationToken));
```

Comment:
Thanks for adding cancellation tokens! ExecuteAsync has an overload that accepts one too, which should prevent it from continuing to retry when cancellation is requested.